web/models/owners.ex

```diff
@@ -1,6 +1,9 @@
 defmodule HexWeb.Owners do
   use HexWeb.Web, :crud
 
+  alias HexWeb.Email
+  alias HexWeb.Mailer
```

Comment:
`HexWeb.Mailer` is already aliased by `use HexWeb.Web, :crud` so this is not needed. Could you add `HexWeb.Email` to list of aliased modules?